md/single-sign-on-with-saml.md

```diff
@@ -0,0 +1,214 @@
+        -->            <PrivateKeyPem>put your private key here</PrivateKeyPem>
+        -->            <CertificatePem>put your certificate here</CertificatePem>
+                   </Key>
+               </Keys>
+               <PrincipalNameMapping policy="FROM_ATTRIBUTE" attribute="username"/>
+               <IDP entityID="idp">
+                   <SingleSignOnService signRequest="true"
+                      validateResponseSignature="true"
+                      requestBinding="POST"
+                      responseBinding="POST"
+        -->           bindingUrl="http://idp.saml.binding.url.to.change"/>
+                   <SingleLogoutService signRequest="false"
+                      signResponse="false"
+                      validateRequestSignature="false"
+                      validateResponseSignature="false"
+                      requestBinding="POST"
+                      responseBinding="POST"
+        -->           postBindingUrl="http://idp.saml.binding.url.to.change"
+        -->           redirectBindingUrl="http://idp.saml.binding.url.to.change"/>
+                   <Keys>
+                       <Key signing="true">
+        -->            <CertificatePem>put your certificate here</CertificatePem>
+                       </Key>
+                   </Keys>
+               </IDP>
+            </SP>
+       </keycloak-saml-adapter>
+   ```
+
+
+#### SAML SSO and Java client application
+
+To enable a Java client application to access the engine using SAML autentication,  
+the simplest way is to use the [`LoginAPI`](http://documentation.bonitasoft.com/javadoc/api/${varVersion}/org/bonitasoft/engine/api/LoginAPI.html#login(java.util.Map)).
+
+### Troubleshoot
+
+To troubleshoote SSO login issues, you need to increase the [log level](logging.md) to `ALL` in order for errors to be displayed in the log files (by default, they are not).
+
+## Configure logout behaviour
+
+#### Bonita BPM Portal
+
+If you are using SAML, when users log out of Bonita BPM Portal, they do not log out of SAML. Therefore they are not logged out of all applications that are using the SAML service.  
+To avoid this, you can hide the logout option of the portal. 
+To do this, set the `logout.link.hidden=true` option in `authenticationManager-config.properties` located in `platform_conf/initial/tenant_template_portal` 
+for not initialized platform or `platform_conf/current/tenant_template_portal` and `platform_conf/current/tenants/[TENANT_ID]/tenant_portal/`.
+
+## Manage passwords
+
+When you are using SAML, the password for a user is managed in your SAML system. However, when you create a user in Bonita BPM Portal, specifying a password is mandatory. This password is ignored. 
+
+## LDAP synchronizer and SAML
+
+If you are using an LDAP service and the [LDAP synchronizer](ldap-synchronizer.md) to manage your user data, you can continue to do this and use SAML. The LDAP synchronizer user must be registered in SAML. 
```

Comment:
and manage authentication over SAML.
The LDAP synchronizer user must be registered in Bonita BPM (no need for an SAML IdP account).